esy-lib/Tarball.re

```diff
@@ -67,6 +67,31 @@ let run = cmd => {
   };
 };
 
+let fixFilePermissionsAfterUnTar = out => {
+  let umask = lnot(System.getumask());
+  Fs.traverse(
+    ~f=
+      (p, s) => {
+        let p = EsyBash.normalizePathForCygwin(Path.show(p));
+        try%lwt(
+          switch (s.st_kind) {
+          | Unix.S_LNK
+          | Unix.S_REG =>
+            let%lwt () = Lwt_unix.chmod(p, s.st_perm lor 0o644 land umask);
```

Comment:
We decided against touching regular files. Didn't we?

Reply:
Done, I totally forgot about this, thanks for catching this